<launch>
  <arg name="result_folder" value="/home/lch/Downloads/sample_data/sgslam/output/online_mapping"/>
  <arg name="agentA_scene" value="ab0201_03a"/>
  <arg name="cfg_file" value="$(find sgloop_ros)/../../../config/realsense.yaml"/>

  <group ns="viz">
    <param name="edge_width" value="0.03"/>
    <param name="edge_color/r" value="0.0"/>
    <param name="edge_color/g" value="1.0"/>
    <param name="edge_color/b" value="1.0"/>
    <param name="centroid_size" value="0.3"/>
  </group>

  <node pkg="sgloop_ros" name="agentA" type="RenderNode" output="screen" clear_params="true">
      <param name="agent_name" value="agentA"/>
      <param name="cfg_file" value="$(arg cfg_file)"/>
      <param name="sequence_result_folder" value="$(arg result_folder)/$(arg agentA_scene)"/>

  </node>

  <node pkg="tf" type="static_transform_publisher" name="link_1st_agent" args="0 0 0 0 0 0 1 world agentA 100" />

</launch>
